Sources/BraintreePayPal/BTPayPalDriver.m

```diff
@@ -178,10 +257,21 @@ - (void)tokenizePayPalAccountWithPayPalRequest:(BTPayPalRequest *)request comple
 
             [self sendAnalyticsEventForInitiatingOneTouchForPaymentType:request.paymentType withSuccess:analyticsSuccess];
 
-            [self handlePayPalRequestWithURL:approvalUrl
-                                       error:error
-                                 paymentType:request.paymentType
-                                  completion:completionBlock];
+            BOOL isPayPalCheckoutRequest = [self.payPalRequest isKindOfClass:BTPayPalCheckoutRequest.class];
+            BOOL shouldUseNativeCheckout = ((BTPayPalCheckoutRequest *)self.payPalRequest).shouldUseNativePayPalCheckout;
```

Comment:
This will crash for requests for the Vault flow (`BTPayPalVaultRequest`). Should be an easy fix though by just adding a check 👍 . 

Maybe something like:
```
BOOL isPayPalCheckoutRequest = [request isKindOfClass:BTPayPalCheckoutRequest.class];
if (isPayPalCheckoutRequest && ((BTPayPalCheckoutRequest *)request).shouldUseNativePayPalCheckout) {
    // do stuff
}
```

Reply:
Ah good call. Thanks!